go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/nspcc-dev/neofs-contract v0.16.0
 	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.7
 	github.com/nspcc-dev/tzhash v1.6.1
+	github.com/olekukonko/tablewriter v0.0.5
```

Comment:
We already have `text/tabwriter`, do we need a new dependency? Or is it for vertical bars?

Reply:
Yes, it is for vertical bars.